src/components/Pf/PfColors.tsx

```diff
@@ -86,13 +86,25 @@ export enum PfColors {
   Red = '#cc0000', // Red100
 
   // Kiali colors that use PF colors
-  Gray = '#72767b' // Black600
+
+  Gray = Black600,
+  GrayBackground = Black150,
+
+  // Health/Alert colors https://www.patternfly.org/v4/design-guidelines/styles/colors
+  Danger = '#C9190B',
+  DangerBackground = '#470000',
```

Comment:
`Red400`

Reply:
Ah, yeah, I copied the hex from the UX page and forgot to match them with he constants they represent.  I'll update.